src/components/Dropdown.jsx

```diff
@@ -52,7 +52,10 @@ class Dropdown extends React.Component {
                                     key={d.id}
                                     onMouseDown={e => this.handleChange(e, d)}
                                 >
-                                    {d.name || d.first_name}
+                                    {d.name === undefined &&
+                                    d.first_name === undefined
+                                        ? d
+                                        : d.name || d.first_name}
```

Comment:
Can you do this computation above the return statement, and maybe simplify it a little bit with readability in mind?